utils/general_config.json

```diff
@@ -0,0 +1,219 @@
+
+//generalConfig components
+
+{
+"debeziumConfig": { 
+ "debezium-mysql" : "{{database.hostname}}",
```

Comment:
@gitcliff  well done on this. we had actually agreed to rather pass args via commandline flags ..
after this Pr is merged https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/pull/54 ,
we wi;; have to revist this

Reply:
sure @mozzy11 , i will refactor this after the PR is merged

Reply:
Thanks @gitcliff for this; I will take a look once #54 is merged and this is updated, if you don't mind.

Reply:
its cool

Reply:
@bashir2  the PR was merged , you can kindly have a look this one to . Thanks

Reply:
Thanks for the reminder. Actually, looking at it now, it is not clear to me how you are using these settings in the code. It seems to me that you have only added a new json config file but no code changes to use its content. Am I missing something?

Reply:
your right @bashir2  ,,,
i have made some changes in the code ...kindly review